buildkite/resource_pipeline_schedule.go

```diff
@@ -38,7 +39,7 @@ func NewPipelineScheduleResource() resource.Resource {
 	return &pipelineSchedule{}
 }
 
-func (ps *pipelineSchedule) Metadata(_ context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
```

Comment:
Nice pickup :)

Reply:
This can be an unnamed; it isn't being referenced in the code so `_` is an acceptable way to use it, it's arguably the correct way to use it, too.

Reply:
@mcncl , @james2791 , it can be unnamed, but was looking at more of a "readability" point of view in the code..